### DISCUSSION TITLE
Strapi and SendGrid

### DISCUSSION THREAD

**Discussion by Mikeez** [strapi/strapi#5636] [Legacy Topics]
Hello,

I having issues with the Email Plugin. I installed the SendGrid email plugin following the Docs

**Describe the bug**
I am getting an internal server error on postman when I do the following: 

`POST: dc:1337/email`
`RAW Body: {
    "to": "mikeez@lol.com",
    "text": "lol",
}`

**Steps to reproduce the behavior**
1. Installed SendGrid Plugin
2. Make an account at SendGrid
3. Change Env setting to SendGrid, add the JWT Token there and Save
4. Try to send an email and Internal server Error

**Expected behavior**
Since I am using Postman, I expect to have a 2xx code. 

**Screenshots**
![image](https://user-images.githubusercontent.com/26636051/77830994-ccea4580-7102-11ea-8827-70f308c94759.png)

**System**
- Node.js version: 10.19
- NPM version: 6.14.4
- Strapi version: 19.0 Beta
- Database: MySQL Latest
- Operating system: Linux CentOS

**Additional context**

I have React and I tried to do an Axios POST request the same way I am doing it on PostMan and I get the following:

`Access to XMLHttpRequest at 'myStrapi:1337/email' from origin 'myStrapi' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
createError.js:16 Uncaught (in promise) Error: Network Error
    at e.exports (createError.js:16)
    at XMLHttpRequest.p.onerror (xhr.js:69)`

**Comment by lauriejim** [comment]
Hello, it looks to be a CORS issue.
Can you make sure you `security.json` in production mode allow CORS.

**Comment by Mikeez** [comment]
> Hello, it looks to be a CORS issue.
> Can you make sure you `security.json` in production mode allow CORS.

```
{
  "csp": {
    "enabled": true,
    "policy": ["block-all-mixed-content"]
  },
  "p3p": {
    "enabled": false,
    "value": ""
  },
  "hsts": {
    "enabled": true,
    "maxAge": 31536000,
    "includeSubDomains": true
  },
  "xframe": {
    "enabled": true,
    "value": "SAMEORIGIN"
  },
  "xss": {
    "enabled": true,
    "mode": "block"
  },
  "cors": {
    "enabled": true,
    "origin": "*"
  },
  "ip": {
    "enabled": false,
    "whiteList": [],
    "blackList": []
  }
}
```


This is my security in Strapi. I do have CORS enabled

**Comment by derrickmehaffy** [comment]
This should no longer be a problem with the stable release, going to mark this as answered as the question is over 100 days old.
